.gitattributes

```diff
@@ -0,0 +1,3 @@
+.github export-ignore
+tests export-ignore
+phpunit.xml.dist export-ignore
```

Comment:
You should include `.gitignore` and `.gitattributes` itself here too :)

Reply:
done